### ISSUE TITLE
[Spigot 1.21] Spamming "Cannot invoke fr.skytasul.glowingentities.GlowingBlocks.unsetGlowing" although glowing-blocks set to false

### ISSUE THREAD

**Issue by hohoaisan** [Norbit4/TreeCuter#3]
As in the title, I know there was update logs in Spigot page about glowing feature being updated for latest version 1.20.6.
So I decided to disable glowing feature knowing ahead that it won't work, but didn't expect it spamming server console when user cutting trees
```
messages:
  permission: '&c&l» &7You do not have permission to use this command!'
  console: '&c&l» &7Only players can use this command!'
  help:
    - '&8[&m------------------------------------------------]'
    - ''
    - ' &8&l» &8/&atreecuter reload &8- &7reload plugin!'
    - ' &8&l» &8/&atreecuter get &8- &7get tool!'
    - ' &8&l» &8/&atreecuter toggle &8- &7on/off!'
    - ''
    - '&8[&m------------------------------------------------]'
  reload:
    start: '&e&l» &7Reloading plugin...'
    end: '&a&l» &7Plugin reloaded!'
  tool:
    get: '&a&l» &7You got tool!'
    disabled: '&c&l» &7Tool is disabled! Enable it in config!'
  toggle:
    enable: '&a&l» &7Enabled cutting down trees!'
    disable: '&c&l» &7Disabled cutting down trees!'

# if true, only players with permission can cut down trees
use-permissions: false
permission: 'treecuter.use'

# max blocks to cut at once
# only important for GIANT custom trees!
max-blocks: 100
# min blocks to cut at once
min-blocks: 4

# if true, the player must hold shift to cut down trees
shift-mining: true # true <- recommended

# apply mining fatigue effect when player is cutting down trees
# works only with shift-mining enabled!
mining-effect:
  enable: true # true <- recommended
  default-level: 2

# if true, blocks that have been cut off, will glow

# [!]                                                                                                         [!]
# [!] GLOWING - SUPPORTS ONLY PAPER OR FORKS VERSIONS: 1.17.1, 1.18.2, 1.19.4, 1.20.2, 1.20.4, 1.20.5, 1.20.6 [!]
# [!]                                                                                                         [!]                                                                                    [!]

glowing-blocks: false
# colors: black, dark_blue, dark_green, dark_aqua, dark_red, dark_purple, gold, gray,
#         dark_gray, blue, green, aqua, red, magenta, light_purple, yellow, white
glowing-color: 'red'

# [!]                                                                                                         [!]
# [!] GLOWING - SUPPORTS ONLY PAPER OR FORKS VERSIONS: 1.17.1, 1.18.2, 1.19.4, 1.20.2, 1.20.4, 1.20.5, 1.20.6 [!]
# [!]                                                                                                         [!]

# cut blocks automatically go to player inventory
items-to-inventory: false

# automatically plants saplings that lie on the ground!
auto-plant: true

custom-tool:
  enable: false
  # material support ItemsAdder, when you use item from ItemsAdder, you must use 'ia:<ID>' instead of 'MATERIAL'
  # example: 'ia:example_item_id'
  material: 'NETHERITE_AXE'
  name: '&c&lGOD AXE'

# blocked worlds
block-worlds:
  - 'example_world'

accept-tools:
#  - 'WOODEN_AXE'
#  - 'STONE_AXE'
#  - 'IRON_AXE'
  - 'GOLDEN_AXE'
  - 'DIAMOND_AXE'
  - 'NETHERITE_AXE'

accept-wood-blocks:
  - 'OAK_LOG'
  - 'SPRUCE_LOG'
  - 'BIRCH_LOG'
  - 'JUNGLE_LOG'
  - 'ACACIA_LOG'
  - 'DARK_OAK_LOG'
  - 'MANGROVE_LOG'
  - 'MANGROVE_ROOTS'
  - 'CHERRY_LOG'
  - 'OAK_WOOD'
  - 'SPRUCE_WOOD'
  - 'BIRCH_WOOD'
  - 'JUNGLE_WOOD'
  - 'ACACIA_WOOD'
  - 'DARK_OAK_WOOD'
  - 'MANGROVE_WOOD'
  - 'CHERRY_WOOD'

auto-plant-saplings:
  - 'OAK_SAPLING'
  - 'SPRUCE_SAPLING'
  - 'BIRCH_SAPLING'
  - 'JUNGLE_SAPLING'
  - 'ACACIA_SAPLING'
  - 'DARK_OAK_SAPLING'
  - 'MANGROVE_PROPAGULE'
  - 'CHERRY_SAPLING'
```

**Comment by Norbit4** [comment]
The plugin does not have support for version 1.21, I have not tested it on this version. I will check it in my free time and make an update.

**Comment by Norbit4** [comment]
Fixed, official support 1.21 has been added
